src/cmd-build

```diff
@@ -119,12 +116,19 @@ if ! git diff --quiet --exit-code; then
 fi
 popd
 commitmeta_input_json=$(pwd)/tmp/commit-metadata-input.json
-cat >"${commitmeta_input_json}" <<EOF
+cat > "${commitmeta_input_json}" <<EOF
 {
   "coreos-assembler.config-gitrev": "${config_gitrev}",
-  "coreos-assembler.config-dirty": ${config_dirty}
+  "coreos-assembler.config-dirty": "${config_dirty}"
 }
 EOF
+
+# Generate an archive metadata about the config GIT checkout, but
+# only if the build is from a Git Recipe.
+if [ -e "/srv/src/config-git" ]; then
```

Comment:
I don't think it's **always** `/srv/src/config-git`. For the https://github.com/coreos/fedora-coreos-config/ configs it's just `srv/src/config`

Reply:
Ack, fixed up in next commit.